### DISCUSSION TITLE
My application for student benefits got rejected

### DISCUSSION THREAD

**Discussion by Shapiro2000** [community/community#136625] [GitHub Education]
### Hi everyone,

I recently applied for the Student pack using my school ID but my application got rejected. Although my student ID does not have any dates on it but since that was the preferred verification, I used it anyway. I have now made another application using my Transcripts (since thats the next best option) and its been 3 days and I have not received any response and I cannot even see the status of this new application. Also, is there an option to upload documents rather than taking pictures as most of my verification documents are in electronic form. This is really urgent as I have tasks with set deadlines to hand in.

**Comment by davevad93** [comment]
Hi @Shapiro2000 ,

You can see you application status [here](https://education.github.com/discount_requests/application) where it says "***You've already submitted X requests***". When I applied last year it was possible to upload the documents but I think it's not possible anymore since I see a lot of people asking for it.

**Comment by aivo9** [comment]
Captain here!

I had the same issue, but I just got approved.

Got rejected multiple times. Time after time. Made few perceptions about how it works, and finally succeeded!

1. Put everything in order in your account settings. Just as the instructions advice you to do. Name, emails, bios, 2 phase authentication, etc.
2. Try to photograph different kind of official school documents with a bad webcam and get rejected multiple times over and over again.
3. Get frustrated.
4. Realize that there is no human approving your entries but artificial intelligence instead "reading" words from the pictures you send.
5. Improvise, adapt, overcome.
6. Open the notebook of your mobile phone and type:

[Your University of Fancy Science]

Student: [insert your name here]
School: [insert your school name (same as in the title) here]
Education: [insert the name of your education here] (I had to add ",Virtual learning" to it cuz I'm not on campus right now, and it got mad)
Study right: [insert the day when you got your study right and the day it ends (in US format e.g. 1 January 2024 - 31 December 2028)]

7. So basically everything is true and you are not lying, but you just made the proof yourself on your mobile phone's notepad.
8. Take a screenshot. Edit the picture with your mobile phone and cut off all the excess space so that the text looks as big as possible in the screen.
9. Open the edited picture on your mobile phone and take the phone so close to your laptop's webcam so that all the words are clearly visible in the picture.
10. Take the photo and check that all the information is visible in the photo.
11. Submit your application.
12. Get approved few minutes later.
13. Not your fault the system of approval is so horrible and broken so feel no shame about cutting the corners.
14. Return the favor and go help others.
15. Enjoy!

Welcome to the community!

*flies away*
